README.md

```diff
@@ -206,6 +218,24 @@ It is also possible to combine several settings into blocks:
 
 The block with several services, ports, etc. will be applied at once. If there is something wrong in the block it will fail as a whole.
 
+```---
+- name: Configure external zone in firewall
+  hosts: myhost
+
+  vars:
+    firewall:
+      - { zone: 'external',
+          service: [ 'tftp', 'ftp' ],
+          port: [ '443/tcp', '443/udp' ],
+          forward_port: [ '447/tcp;;1.2.3.4',
+                          '448/tcp;;1.2.3.5' ],
+          state: 'enabled' }
+  roles:
+    - firewall
```

Comment:
Should use the FQRN - `linux-system-roles.firewall`

Reply:
Fixed